src/CMakeLists.txt

```diff
@@ -3,4 +3,6 @@ add_subdirectory(image_view)
 if (CATKIN_ENABLE_TESTING)
   add_subdirectory(test)
 endif()
-add_subdirectory(python_bindings)
+if (QT_VERSION_MAJOR EQUAL 5)
+  add_subdirectory(python_bindings)
+endif()
```

Comment:
What are the major issues with providing python support?

Reply:
Ubuntu 22.04 does not include a Qt6 version of pyqt/pyside so I was not able to compile `python-qt-binding`